pctasks/core/pctasks/core/logging.py

```diff
@@ -267,11 +268,13 @@ def from_task_run_config(
         package: Optional[str] = None,
         level: int = logging.INFO,
         log_format: str = DEFAULT_TASK_LOG_FORMAT,
+        client_secret_credentials: Optional[ClientSecretCredentials] = None,
```

Comment:
This ensures that the TaskIO SP can write the final logs to Blob Storage when the task finishes.